app/lib/validators/user_validator.rb

```diff
@@ -9,6 +9,6 @@ class UserValidator < ActiveModel::Validator
   def validate(record)
```

Comment:
would it be worth changing the allow? method to just accept the record and then maybe use the splat operator?

Reply:
yes I agree, I will check all the records as in some cases like the locations_controller, the action and controller were not being permitted through making any validation redundant!